lib/presentation/mixins/media_viewer_app_bar_mixin/media_viewer_app_bar_mixin.dart

```diff
@@ -29,17 +31,44 @@ mixin MediaViewerAppBarMixin {
     Event? event,
   ) async {
     Matrix.of(context).shareContent = event?.content;
-    final result = await showDialog(
-      context: context,
-      useSafeArea: false,
-      useRootNavigator: false,
-      builder: (c) => const Forward(),
-    );
+    final responsive = getIt.get<ResponsiveUtils>();
+
+    final result = responsive.isMobile(context)
+        ? await _showForwardMobileDialog(context)
+        : await _showForwardWebDialog(context);
+
     if (result is PopResultFromForward) {
       Navigator.of(context).pop<PopResultFromForward>();
     }
   }
 
+  Future<PopResultFromForward?> _showForwardMobileDialog(
+    BuildContext context,
+  ) async =>
+      await showDialog(
+        context: context,
+        useSafeArea: false,
+        useRootNavigator: false,
+        builder: (c) => const Forward(),
+      );
```

Comment:
- any impaction if it is dialog in mobile? why not keep as before?
- please demo also the case `share file from other app to twake chat`

Reply:
I've not seen any impaction on mobile. It is basically exactly the same as before but in a method to handle separately the mobile and web case line 36.

The videos have been updated in this PR's description. There's a demo of the case `share file from other app to twake chat`